src/breadcrumbs/previews/example.data.json

```diff
@@ -1,3 +1,29 @@
 {
-  "colour": " alt"
+  "brandOverride": "alt",
+  "breadcrumbs": [
+    {
+      "link": "#",
```

Comment:
Could update to have the real data that's on forgov for a more lifelike example.